VerbalExpressions/VerbalExpressions.swift

```diff
@@ -21,84 +21,132 @@ public class VerbalExpressions {
 
     // computed properties
     var pattern: String { return prefixes + source + suffixes }
-    
+
     var regularExpression: NSRegularExpression! {
-        get {
-            let regex = try! NSRegularExpression(pattern: pattern, options: options)
-            
-            return regex
-        }
+        return try! NSRegularExpression(pattern: pattern, options: options)
     }
 
     // instance methods
-    public func startOfLine(enabled enabled: Bool = true) -> Self {
-        prefixes = enabled ? "^" : ""
-
-        return self
+    public func startOfLine(enabled enabled: Bool = true) -> VerbalExpressions {
+        return VerbalExpressions(
+            prefixes: enabled ? "^" : "",
+            source: self.source,
```

Comment:
Not seen it mentioned either way. `self.` just feels a little more explicit to me. I don't have any strong feeling about that however, I'll clean them up.